Source/EditorTab.cpp

```diff
@@ -895,6 +896,32 @@ void EditorTab::AddCollision()
     mUndoStack.push(new AddCollisionCommand(this));
 }
 
+void EditorTab::ConnectCollisions()
+{
+    if (!mScene->selectedItems().isEmpty())
+    {
+        std::vector<ResizeableArrowItem*> collisions;
+
+        for (auto& selectedItem : mScene->selectedItems())
+        {
+            auto asResizableArrowItem = dynamic_cast<ResizeableArrowItem*>(selectedItem);
```

Comment:
use qgraphics_itemcast or whatever it is

Reply:
changed to ``qgraphicsitem_cast``